### PR TITLE
Update typings.d.ts

### PR DESCRIPTION
**Pull request by dlahish**
Update typings in `offlineActionTracker` actions arguments

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "flow:stop": "flow stop",
     "flow": "flow; test $? -eq 0 -o $? -eq 2",
     "lint": "eslint src/",
-    "prepublish": "npm run lint && npm run flow && npm run test && npm run build",
```
Comment (sorodrigo) [inline package.json]: I think you didn't wanted to commit this.

**Review by sorodrigo — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (calumpeak) [reply package.json]: Agreed.

**Review by calumpeak — CHANGES_REQUESTED** [review]
Please revert prepublish step.

**Comment by dlahish** [comment]
Sorry, I didn't want to commit this

**Comment by calumpeak** [comment]
@dlahish thanks for raising this, we're going to go with #244 which came in at the same time but has a cleaner commit history and the commit message is more informative.

**Pull request closed without merging**